src/app/legacy/containers/PageHandlers/withOptimizelyProvider/index.jsx

```diff
@@ -14,6 +14,7 @@ import { ServiceContext } from '../../../../contexts/ServiceContext';
 import isCypress from './isCypress';
 
 const isInCypress = isCypress();
+const TIMEOUT_INTERVAL = 5000;
```

Comment:
We should keep an eye on CWV by changing this, it should be fine as you've mitigated with a placeholder, but would be a good thing to bare in mind.